EASI.postman_collection.json

```diff
@@ -3020,12 +3059,12 @@
 		},
 		{
 			"key": "CedarApiKey",
-			"value": "",
+			"value": "7b31e270-2373-4908-9dec-3a105efbd2ea",
```

Comment:
Oops? Do they need to rotate this now?

Reply:
Oooof - sorry. That isn't good. I left that in postman. Yes they probably should rotate that, and I'll let people know to update locally. Should I ping CEDAR in their slack?

Nice catch - thank you